### DISCUSSION TITLE
[Question]  how to reset password phpmyadmin

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#13918] [How to] [answered]
is it possible to reset admin password by phpmyadmin, if yes how ? 
forgot password dont work as emails are not being sent..

**Comment by vovpff** [comment]
There are reset password script available in scripts repo.

**Comment by mariuszkrzaczkowski** [comment]
describe in more detail, do you want to reset the password of the CRM user or the password of the database user?

**Comment by vovpff** [comment, marked as answer]
@rioshaz look here https://github.com/YetiForceCompany/YetiForceScripts/tree/master/PasswordReset
